### PR TITLE
Fat 1703/C353639

### PR DESCRIPTION
**Pull request by kimkate303**
https://foliotest.testrail.io/index.php?/cases/view/353639

### CODE REVIEW

File: cypress/integration/inventory/search/browse-contributors-exact.spec.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+describe('Search: browse contributors with exact match query', () => {
+  const testData = {};
+  const instance = {
+    source: 'FOLIO',
+    title: `Test_title_A_${getRandomPostfix()}`,
+    contributors: [
+      {
+        name: `Test_contributor_A_${getRandomPostfix()}`,
+        contributorNameTypeId: '2b94c631-fca9-4892-a730-03ee529ffe2a',
```
Comment (IhorBohdan) [inline cypress/integration/inventory/search/browse-contributors-exact.spec.js:18]: It's hardcode id.Can we use uiid()?

File: cypress/support/fragments/inventory/inventorySearch.js [hunk 2]
```diff
@@ -63,12 +63,14 @@ export default {
...
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
     cy.wait(1000);
     cy.do(Select('Search field index').choose('Browse call numbers'));
   },
 
   selectBrowseSubjects() {
     // cypress can't draw selected option without wait
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
     cy.wait(1000);
     cy.do(Select('Search field index').choose('Browse subjects'));
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/inventorySearch.js:75]: Move to const Select

File: cypress/integration/inventory/search/browse-contributors-exact.spec.js (hunk 1 above)
Comment (kimkate303) [reply cypress/integration/inventory/search/browse-contributors-exact.spec.js:18]: yes) sorry it is actually in draft status

File: cypress/support/fragments/inventory/inventorySearch.js (hunk 2 above)
Comment (kimkate303) [reply cypress/support/fragments/inventory/inventorySearch.js:75]: ok, thanks

File: cypress/support/fragments/inventory/search/browseContributors.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    cy.do(MultiColumnListCell(contributor.name).has({ innerHTML: `<strong>${contributor.name}</strong>` }));
+    // TODO add check for contributor.contributorTypeText
+  },
+  openInstance(contributor) {
+    cy.do(MultiColumnListCell(contributor.name).click());
+  },
+  checkInstance(instance) {
+    cy.expect([
+      // TODO: add check for date with format <C6/8/2022, 6:46 AM>
+      PaneContent({ id: 'pane-instancedetails-content' }).find(KeyValue({ value: instance.title })).exists(),
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/search/browseContributors.js:50]: Move PaneContent to const

File: cypress/support/fragments/inventory/search/browseContributors.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+      Form().find(Button({ id: 'accordion-toggle-button-nameType' })).exists(),
+      // TODO add check for Relator term accordeon button
+      Section({ id: 'pane-results' }).find(Heading('Browse inventory')).exists(),
+      PaneHeader({ id: 'paneHeaderpane-results' }).find(HTML('Enter search criteria to start browsing')).exists(),
+      PaneContent('Browse for results entering a query or choosing a filter.').exists(),
+    ]);
+  },
+  browse(contributorName) {
+    cy.do([
+      TextInput({ id: 'input-inventory-search' }).fillIn(contributorName),
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/search/browseContributors.js:35]: Move to const

File: cypress/support/index.js [hunk 5]
```diff
@@ -18,7 +18,7 @@ import './data-import';
 import './commands';
 
 
-setInteractorTimeout(20_000);
+setInteractorTimeout(30_000);
```
Comment (IhorBohdan) [inline cypress/support/index.js:21]: You need ask another part of team to change this file

Comment (kimkate303) [reply cypress/support/index.js:21]: thanks for advice

Comment (kimkate303) [reply cypress/support/index.js]: changed to 80K)

**Review by TetianaParanich — APPROVED** [review]

File: cypress/integration/inventory/search/browse-contributors-exact.spec.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
+import permissions from '../../../support/dictionary/permissions';
+import inventorySearch from '../../../support/fragments/inventory/inventorySearch';
+import browseContributors from '../../../support/fragments/inventory/search/browseContributors';
+import topMenu from '../../../support/fragments/topMenu';
+import users from '../../../support/fragments/users/users';
```
[the remark this answers is not in the document]
Comment (kimkate303) [reply cypress/integration/inventory/search/browse-contributors-exact.spec.js]: changed

**Review by dmitriyfolio — APPROVED** [review]

**Pull request merged**